[DMS-1254] Add Ed-Fi API v8.0  announcement and documentation updates (#554)
Ed-Fi-Alliance-OSS/ed-fi-alliance-oss.github.io: blog/2026-07-15.md

---
title: '📢 Ed-Fi API v8.0: Now Available'
authors: [mayya]
tags: [releases, ed-fi-api, dms]
date: 2026-07-15
---

The Ed-Fi Alliance is excited to announce the release of **Ed-Fi API v8.0** and
**MetaEd v4.8**. Ed-Fi API v8.0 represents the next-generation evolution of the
Ed-Fi ODS/API platform, delivering a modernized architecture while maintaining

> compatibility with existing Ed-Fi API integrations. v8.0 is designed as the

Reviewer: ```suggestion
compatibility with existing Ed-Fi API integrations. Version 8 is designed as the
```

long-term replacement for the legacy ODS/API platform.

Have questions about the transition from ODS/API to Ed-Fi API v8? See the [Ed-Fi
ODS/API and Ed-Fi API v8 FAQ](/reference/roadmap/api-faq) for answers on the
roadmap, timelines, and what to expect.

<!-- truncate -->

## Ed-Fi API v8.0

Ed-Fi API v8.0 is a new platform, not an incremental update to the ODS/API. It
introduces two new services that take over the roles currently filled by the

> Ed-Fi ODS/API and Ed-Fi Admin API:

Reviewer: ```suggestion
Ed-Fi ODS/API and Ed-Fi ODS Admin API:
```

   storage model stores each Ed-Fi resource in its own set of dedicated
   relational tables, organized into a schema per model project.
2. **Multi-Tenancy and Data Segmentation**: The Configuration Service manages
   data stores and tenant context. Multiple tenants can share a single API
   deployment with complete data isolation enforced at the database level.
3. **Profiles**: Readable and writable Profiles are supported, allowing
   implementers to define data policies that restrict the resources and
   properties visible to a specific API client application.
4. **Changed Record Queries**: This feature allows client systems to narrow
   requests for data to only data that has changed since a specified change
   version.
5. **Self-Contained Identity Provider**: A built-in OpenIddict-based identity
   provider is hosted inside the Configuration Service, so no external identity
   server is required for a standard deployment. Keycloak remains supported as
   an alternative.
6. **Docker-Based Deployment**: Ed-Fi API v8.0 is distributed as Docker images.
   Startup scripts and a reference `docker-compose.yml` are provided for local
   deployment.

[What's New in Ed-Fi API

> v8.0](https://docs.ed-fi.org/reference/ed-fi-api/8.0/whats-new/whats-new-in-this-release)

Reviewer: ```suggestion
v8.0](/reference/ed-fi-api/8.0/whats-new/whats-new-in-this-release)
```

covers each of these in detail, and the [Breaking

> Changes](https://docs.ed-fi.org/reference/ed-fi-api/8.0/whats-new/release-notes#ed-fi-api-v80---breaking-changes-from-previous-releases)

Reviewer: ```suggestion
Changes](/reference/ed-fi-api/8.0/whats-new/release-notes#ed-fi-api-v80---breaking-changes-from-previous-releases)
```

section of the release notes has the full migration checklist.

## MetaEd v4.8

MetaEd v4.8 adds support for Ed-Fi API v8.0, including ApiSchema NuGet packaging
as asset-only content packages for use by the Ed-Fi API. It continues to support
all currently supported versions of the Ed-Fi Data Standard and ODS/API.
Implementers extending Ed-Fi API v8.0 will need to update to MetaEd IDE v4.8.

[MetaEd 4.8 Release

> Notes](https://docs.ed-fi.org/reference/metaed/releases/4.8.0)

Reviewer: ```suggestion
Notes](/reference/metaed/releases/4.8.0)
```

---

## Try It Out

Follow the instructions in the [Getting

> Started](https://docs.ed-fi.org/reference/ed-fi-api/8.0/getting-started)

Reviewer: ```suggestion
Started](/reference/ed-fi-api/8.0/getting-started)
```
